### PR TITLE
GP2-759_Product_search_full_scroll

### PR DESCRIPTION
**Pull request by bobmeredith**
## To do (delete all that do not apply):

 - [X] Ticket exists in Jira  https://uktrade.atlassian.net/browse/TICKET_ID_HERE 
 - [ ] Jira ticket has the correct status.
 - [X] [Changelog](CHANGELOG.md) entry added.

 - [ ] (if is a UI change) Includes screenshot(s) - ideally before and after, but at least after

 
 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
image

File: react-components/src/components/ProductFinder/ProductFinder.jsx [hunk 1]
```diff
@@ -20,30 +20,74 @@ const formatPath = (pathstr) => {
...
+    setSearchResults()
   }
 
   const closeConfirmation = () => {
     setProductConfirmationRequired(false)
     openProductFinder(true)
   }
 
+  const modalAfterOpen = () => {
+    setIsScrolled({})
```
Comment (lewis-coulson-dit) [inline react-components/src/components/ProductFinder/ProductFinder.jsx:65]: How come the empty object as argument?

**Review by lewis-coulson-dit — APPROVED** [review]

File: react-components/src/components/ProductFinder/ProductFinder.jsx (hunk 1 above)
Comment (bobmeredith) [reply react-components/src/components/ProductFinder/ProductFinder.jsx:65]: empty object - was a bool, but I wanted a scroll blur at the top and bottom working independently, so became an object.

**Pull request merged**